### DISCUSSION TITLE
[BUG]Elsa 有没有QQ群？

### DISCUSSION THREAD

**Discussion by zqhgit001** [elsa-workflows/elsa-core#5469] [Q&A]
您好：Elsa有没有QQ群？qq群号是多少？

**Comment by glime-ai** [comment]
很抱歉，根据提供的信息，我无法找到关于Elsa Workflow是否有官方QQ群及其群号的信息。通常这类信息会在项目的官方网站、官方论坛、GitHub页面或者社交媒体上公布。建议您查看Elsa Workflow的官方网站或者GitHub页面，或者在相关技术社区和论坛中询问以获取这方面的信息。如果您需要，我可以帮助您查找这些资源的链接。

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).

**Comment by sfmskywalker** [comment]
我们目前没有一个 QQ 群。如果您有其他问题或需要帮助，请随时在 GitHub 讨论区发帖，我们会尽力提供支持。

We currently do not have a QQ group. If you have any other questions or need assistance, please feel free to post in the GitHub discussions, and we will do our best to provide support.
